### PR TITLE
Update forging example (docker) - Closes #103

### LINKED ISSUE #103: Update forging example for Docker
Related PR: https://github.com/LiskHQ/lisk/pull/2521/files

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #103

### CODE REVIEW

File: user-guide/administration/docker/admin-docker.md [hunk 1]
```diff
@@ -45,5 +45,5 @@ Configuration variables always start with `LISK_` and are mapped to their path i
 
 Info | Note 
 --- | --- 
-![info note](../../../info-icon.png "Info Note") | Adding an array component as environment variable in your docker compose is a bit tricky. For example, if you aim to enable forging on your node, you need to add the next variable which will be inserted as the **first** component in the array `forging.delegates`:`- LISK_FORGING_DELEGATES_1=<encrypted_passphrase>:<publickey>`
+![info note](../../../info-icon.png "Info Note") | Adding an array component as environment variable in your docker compose is a bit tricky. For example, if you aim to enable forging on your node, you need to add the next variable which will be inserted as the **first** component in the array `forging.delegates`:- LISK_FORGING_DELEGATES_1=publicKey1&#124;encryptedPassphrase1,publicKey2&#124;encryptedPassphrase2</code>
```
Comment (diego-G) [inline user-guide/administration/docker/admin-docker.md]: Please, check again the formatting.

**Review by diego-G — CHANGES_REQUESTED** [review]

File: user-guide/administration/docker/admin-docker.md [hunk 2]
```diff
@@ -14,40 +17,48 @@ Command | Description
...
 docker restart container_id
 ```
 
 ## Configuration
 
-Lisk Core normally gets configured by providing a suited [configuration file](../../configuration/configuration.md). But this isn't convenient in a Docker installation. Instead a [template](https://github.com/LiskHQ/lisk/blob/1.1.0/docker_files/etc/confd/templates/config.json.tmpl) is provided and configurations options can be passed as environment variables.  We did so already by setting `LISK_DB_HOST` in `docker-compose.yml` above.
+Lisk Core normally gets configured by providing a suited [configuration file](../../configuration/configuration.md).
+But this isn't convenient in a Docker installation.
+Instead, one can make use of the environmental variables to change values of configuration options.
+To do so, open the `.env` file, that you created during [docker installation](../../setup/install/docker/installation-docker.md), and adjust the values to your needs.
```
Comment (diego-G) [inline user-guide/administration/docker/admin-docker.md]: Link doesn't work.

File: user-guide/administration/docker/admin-docker.md [hunk 3]
```diff
@@ -14,40 +17,48 @@ Command | Description
...
+```bash
 docker restart container_id
 ```
 
 ## Configuration
 
-Lisk Core normally gets configured by providing a suited [configuration file](../../configuration/configuration.md). But this isn't convenient in a Docker installation. Instead a [template](https://github.com/LiskHQ/lisk/blob/1.1.0/docker_files/etc/confd/templates/config.json.tmpl) is provided and configurations options can be passed as environment variables.  We did so already by setting `LISK_DB_HOST` in `docker-compose.yml` above.
+Lisk Core normally gets configured by providing a suited [configuration file](../../configuration/configuration.md).
+But this isn't convenient in a Docker installation.
+Instead, one can make use of the environmental variables to change values of configuration options.
```
Comment (diego-G) [inline user-guide/administration/docker/admin-docker.md]:
```suggestion
Instead, one can use environment variables to change values in the configuration.
```

**Review by diego-G — CHANGES_REQUESTED** [review]

File: user-guide/administration/docker/admin-docker.md (hunk 3 above)
Comment (diego-G) [reply user-guide/administration/docker/admin-docker.md]: Please check this across the file. There are several cases.

File: user-guide/administration/docker/admin-docker.md (hunk 2 above)
Comment (Tschakki) [reply user-guide/administration/docker/admin-docker.md]: fixed

File: user-guide/administration/docker/admin-docker.md [hunk 4]
```diff
@@ -14,40 +17,48 @@ Command | Description
...
+These can be changed directly in the file, if needed.
+For example, the value of `redis.db.host` can be changed by setting the `LISK_REDIS_DB_HOST` environment variable.
+If you are unsure what variable name to use, refer to the [list of command line options](../source/admin-source.md#command-line-options).
 
 Info | Note 
 --- | --- 
-![info note](../../../info-icon.png "Info Note") | Adding an array component as environment variable in your docker compose is a bit tricky. For example, if you aim to enable forging on your node, you need to add the next variable which will be inserted as the **first** component in the array `forging.delegates`:`- LISK_FORGING_DELEGATES_1=<encrypted_passphrase>:<publickey>`
+![info note](../../../info-icon.png "Info Note") | Adding an array component as environment variable in your docker compose is a bit tricky. For example, if you aim to enable forging on your node, you need to add the next variable which will be inserted in the array `forging.delegates`: _LISK_FORGING_DELEGATES=publicKey1&#x7c;encryptedPassphrase1,publicKey2&#x7c;encryptedPassphrase2_
```
Comment (diego-G) [inline user-guide/administration/docker/admin-docker.md]:
```suggestion
![info note](../../../info-icon.png "Info Note") | Adding an array component as environment variable in your docker compose is not straightforward. For example, if you aim to enable forging on your node for several delegates, you need to add the next variable which will be inserted in the array `forging.delegates`: _LISK_FORGING_DELEGATES=publicKey1&#x7c;encryptedPassphrase1,publicKey2&#x7c;encryptedPassphrase2_
```

**Review by diego-G — APPROVED** [review]

**Pull request merged**